src/components/GenreSelect/GenreSelect.tsx

```diff
@@ -11,11 +12,11 @@ export function GenreSelect({ selectedGenre, genres, onSelect }: Props): ReactEl
 	return <ul className={styles.genreList}>
 		{genres.map((genre: string, index: number) => {
 				return <li key={index}>
-					<button data-testid="genre-select-button" className={selectedGenre === genre ? styles.active : ''} type="button" onClick={() => {
+					<Button data-testid="genre-select-button" className={selectedGenre === genre ? styles.active : ''} onClick={() => {
```

Comment:
This is use case for `classnames` library (or analogs or your own solution)

Reply:
Done